dev_guide/deployments.adoc

```diff
@@ -131,12 +152,12 @@ shows deployments in the *Browse* tab.
 To cancel a running or stuck deployment:
 
 ----
-$ oc deploy <deployment_config> --cancel
+$ oc deploy --cancel dc/<name>
 ----
 
 WARNING: The cancellation is a best-effort operation, and may take some time to
-complete. It's possible the deployment will partially or totally complete
-before the cancellation is effective.
+complete. It's possible the replication controller will partially or totally complete
+being deployed before the cancellation is effective.
```

Comment:
@kargakis for this case, should we have `oc deploy --cancel --undo` that will make sure when you cancel the deployment it will rollback automatically if it has to?